client/src/components/ChatMessage.tsx

```diff
@@ -2,21 +2,60 @@ import React from "react";
 
 interface Props {
   text: string;
+  userIndex: number;
 }
 
-const messageStyle = {
-  display: "flex",
-  flexDirection: "column" as "column",
-  width: "200px",
-  alignItems: "flex-end",
-  padding: "10px",
-  borderRadius: "10px",
-  backgroundColor: "#09f",
-  margin: "10px"
+const getMessageStyle = (userIndex: number) => {
+  if (userIndex === 0) {
```

Comment:
This is close, but what we really want to do is compare the Message's userIndex (see `src/types.ts`) with the userIndex stored in ChatBox's state. 

We don't know that 0 or 1 necessarily corresponds to blue or white, as we don't know the userIndex of the current user. I.e., if I join the chat second, my userIndex will be 1. Therefore all messages with userIndex 1 should be colored blue. However, if I join the chat first, my userIndex will be 0. Therefore all the messages with userIndex 1 should be colored white.

Reply:
Ah okay, makes sense

Reply:
In order to access the "current user" or "message", should the styling of ChatMessage be outside of the component?